runner/update_status.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+if [ "${RUNNER_CUSTOM_RBAC:-}" == "true" ]; then
+    export HTTPS_PROXY=
+    export https_proxy=
```

Comment:
What is the purpose of these two lines? Do we want to unset them?

Reply:
I wanted to avoid using proxy when running the curl command in case people uses proxies in the pod